### DISCUSSION TITLE
Ignored npm dependency can't be un-ignored

### DISCUSSION THREAD

**Discussion by danewilson** [renovatebot/renovate#32612] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

I ignored an npm dependency in a shared preset via `ignoreDeps`, which worked as intended, but now that I've removed `ignoreDeps` altogether, the dependency is still being ignored.

Things I've tried:
- specifying `"ignoreDeps": []` in the shared preset, instead of removing it
- specifically enabling the package via `packageRules` in the shared preset

No matter what I do, the dependency is still ignored (`@org/package` is a placeholder):
```
{
  "currentValue": "^26.1.1",
  "datasource": "npm",
  "depName": "@org/package",
  "depType": "devDependencies",
  "lockedVersion": "26.1.1",
  "packageName": "@org/package",
  "prettyDepType": "devDependency",
  "skipReason": "ignored",
  "updates": []
}
```

That's the only log that mentions that dep name (including with `printConfig` enabled). So I'm stumped.

Renovate version is 39.19.0.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by danewilson** [comment]
Of course immediately after I post this, `@org/package` is no longer being ignored. Still, it took 2.5 hours for it to be unignored, so it would be nice if there was a way to bust whatever cache is being used here (if I'm correct in assuming it's a caching issue).

**Comment by danewilson** [comment]
Reopening this because `@org/package` is still reported as ignored for multiple repos. Some repos are showing an available update, others are apparently still caching the ignored status for this package somehow. This is well beyond the time I'd expect to wait for this package to be unignored.

**Comment by rarkins** [comment, marked as answer]
I suspect this is related to extract caching. So for any repo where the cache is invalidated (eg via a new commit to the main branch) then you'd see this resolved, and for the rest it will remain. Can you confirm this by committing to one of the repos with this problem?

**Reply by danewilson** [reply]
That does align with what I'm seeing, thanks for your help. Is there a way to invalidate that cache without committing to the main branch?

**Reply by rarkins** [reply]
No

**Comment by rarkins** [comment]
This will be resolved eventually once:
- #33108 is released
- extract cache is updated at least once (e.g. by a commit)

From then on the skipReason will be removed when the cache is being reused
